internal/lsp/command.go

```diff
@@ -1,8 +1,13 @@
 package lsp
 
-import "github.com/styrainc/regal/internal/lsp/types"
+import (
+	"encoding/json"
```

Comment:
Doesn't matter much in this context, but in most of Regal we use this now https://github.com/anderseknert/roast/blob/main/pkg/encoding/json.go